README.md

```diff
@@ -36,23 +36,24 @@ This will start the main nodejs application providing backend service for fronte
 ```shell
 npm start
 ```
-Access find orders API at http://localhost:8080/findAvailableProducts. This is used to demo HTTP stubbing using OpenAPI and Kafka mocking using AsyncAPI<br>
-_*Note:* Unless domain API service and Kafka mocks are running, the above requests will fail. Move to the next section for the solution!_
+Access find orders API at http://localhost:3000/findAvailableProducts. This is used to demo HTTP stubbing using OpenAPI and Kafka mocking using AsyncAPI<br>
+
+_**Note:** Unless domain API service and Kafka mocks are running, the above requests will fail. Move to the next section for the solution!_
 
 ### Start BFF Server with Dependencies (Domain API Stub and Kafka Mock Servers)
 This will start the nodejs based BFF server with domain API stubbed using Specmatic HTTP stub server and Kafka mocked using Specmatic Kafka mock server to demonstrate workings of the stub server
 ```shell
 npm run startWithDeps
 ```
-Access find orders API again at http://localhost:8080/findAvailableProducts with a result like
+Access the find orders API at http://localhost:3000/findAvailableProducts **with a header of pageSize: number**. The result should be like
```

Comment:
Let's keep the application port 8080 itself so that it is consistent across projects